liwo_services/main.py

```diff
@@ -177,6 +177,16 @@ def load_breach_layer():
     result = rs.fetchone()
     return json.dumps(result[0])
 
+@v2.route("/filter_variants", methods=["GET"])
```

Comment:
This is a huge function 🥇
As this is not connected to the database, we're hardcoding information in de Python code?

Also, `filter_properties` from the docstrings seems a better endpoint name?

Reply:
these are properties of variants. You filter variants based on certain properties... We will propably move this next sprint to database. But before adjusting db schema I want to discuss with frontend if this the proper format. So this is a in between solution